src/utils/hooks/useUpdateReviewResponse.ts

```diff
@@ -0,0 +1,35 @@
+import { useUpdateReviewResponseMutation } from '../generated/graphql'
+import getReviewQuery from '../../utils/graphql/queries/getReview.query'
+import { useState } from 'react'
+
+interface UseUpdateReviewResponse {
+  reviewId: number
+  serialNumber: string
+}
+
+const useUpdateRevieResponse = ({ reviewId, serialNumber }: UseUpdateReviewResponse) => {
```

Comment:
Spelling `useUpdateReviewResponse` (think this refers to the same as earlier one)

Reply:
Yes. That is probably how it get spread out. Thanks! Fixing now